cogs/mediaconverter.py

```diff
@@ -18,15 +18,20 @@ async def on_message(self, message: Message):
             return
 
         link_info = self.extract_link(message.content)
-        if link_info[0] is "tiktok":
-            embedded_video = self.embed_tiktok(link_info[1])
-            await message.edit(
-                suppress_embeds=True
-            )  # Removes previous embed from context message
-            await message.channel.send(f"[TikTok Link]({embedded_video})")
-        elif link_info[0] is "twitter":
-            converted_link = self.convert_twitter_link(link_info[1])
-            await message.channel.send(f"[Converted Twitter Link]({converted_link})")
+        if link_info:
+            description = ""
+            if link_info[0] == "tiktok":
+                embedded_video = self.embed_tiktok(link_info[1])
+                description = f"[TikTok Link]({embedded_video})"
+            elif link_info[0] == "twitter":
+                converted_link = self.convert_twitter_link(link_info[1])
+                description = f"[Converted Twitter Link]({converted_link})"
+                
+            if description:     # Added this check to avoid an error about sending empty messages
```

Comment:
This is happening because `self.extract_link` returns `(None, None)` on no match. `(None, None)` is a Tuple object that exists, and thus is True.

To fix this, on the line `link_info = self.extract_link(message.content)`, write it as:

```
link_type, link_url = self.extract_link(message.content)
if link_type:
   ...
```

Because now `link_type` and `link_url` will be assigned as `None` each, and thus `if None` will equal False allowing you to voiding this if check. Also makes the code slightly more readable by having individual variables rather than indexing.